### ISSUE TITLE
imbalanced N fixed BE dataset

### ISSUE THREAD

**Issue by taeil** [Berkeley-Data/irrigation_detection#90]
current code doesn't seem to be easily stop specific N images out of full dataset.  We need training dataset that are imbalanced (for multi-class detection). 

N positive and N negative irrigation (N=64,128,256,512,1024 images)

**Comment by taeil** [comment]
![image](https://user-images.githubusercontent.com/3128981/108807242-b8a00a00-7558-11eb-8ce6-1f96469bdd2d.png)

**Comment by suryatechie** [comment]
"...imbalanced ...."  
"N positive and N negative irrigation (N=64,128,256,512,1024 images)"
@taeil could you please elaborate? If N=64, how many positives and how many negatives?

**Comment by taeil** [comment]
Please hold on. No positive or negative. Ideally same ratios of labels for each class as full datasets. Let me see if I can do from coding on finetune.

**Comment by suryatechie** [comment]
Got it. Thank you.

**Comment by taeil** [comment]
hi @suryatechie I think this would be still valuable for us to run training on separate thread for related work. Can you help create dataset when you have time (not urgent at all)

**Comment by suryatechie** [comment]
Sure @taeil

**Comment by suryatechie** [comment]
Created the dataset. Some of the files might not be required (e.g, 1024 test/val). 
u1024_train.tfrecord
pos: 21
neg: 1003

u512_train.tfrecord
pos: 13
neg: 499

u256_train.tfrecord
pos: 8
neg: 248

u128_train.tfrecord
pos: 4
neg: 124

u64_train.tfrecord
pos: 5
neg: 59

u32_train.tfrecord
pos: 2
neg: 30

u1024_test.tfrecord 
pos: 35
neg: 989

u512_test.tfrecord
pos: 15
neg: 497

u256_test.tfrecord
pos: 11
neg: 245

u128_test.tfrecord
pos: 3
neg: 125

u64_test.tfrecord
pos: 4
neg: 60

u32_test.tfrecord
pos: 1
neg: 31

u1024_val.tfrecord 
pos: 36
neg: 988

u512_val.tfrecord
pos: 11
neg: 501

u256_val.tfrecord
pos: 4
neg: 256

u128_val.tfrecord
pos: 4
neg: 124

u64_val.tfrecord
pos: 1
neg: 62

u32_val.tfrecord
pos: 1
neg: 31

**Comment by suryatechie** [comment]
**S3 location:** s3://agcapbky/BigEarthNet/interim/unbalanced-fixed-count/

**Comment by taeil** [comment]
@taeil  will get to this once other priority tasks are done.
